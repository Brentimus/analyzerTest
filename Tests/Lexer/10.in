3657.35
%11001110111.123
&2755.675
$539.12