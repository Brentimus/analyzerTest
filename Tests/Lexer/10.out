1  1  Double  3.6573499999999999E+003  3657.35
2  1  Double  1.6551230000000000E+003  %11001110111.123
3  1  Double  1.5176750000000000E+003  &2755.675
4  1  Double  1.3371199999999999E+003  $539.12
4  8  Eof